packages/@adobe/spectrum-css-temp/components/accordion/index.css

```diff
@@ -67,16 +66,14 @@ governing permissions and limitations under the License.
   box-sizing: border-box;
   /* left padding takes into account the icon's size as well as the focus state's left border */
   padding-block: var(--spectrum-accordion-item-title-padding-y);
-  padding-inline-start: calc(calc(var(--spectrum-accordion-item-padding) * 2) + 2px);
+  padding-inline-start: 2px;
   padding-inline-end: var(--spectrum-accordion-item-padding);
   margin: 0;
 
   min-height: var(--spectrum-accordion-item-height-actual);
 
-  font-size: var(--spectrum-accordion-item-title-text-size);
+  font-size: var(--spectrum-global-dimension-font-size-100);
   line-height: var(--spectrum-accordion-item-line-height);
-  text-transform: uppercase;
-  letter-spacing: var(--spectrum-accordion-item-title-tracking-actual);
 
```

Comment:
These don't look like the xd file at all, but I guess we don't care for now? Will it be weird if people are using the alpha though?

Reply:
yeah, agreed, i've updated to be closer, including fixing our top border